microsoft-edge/devtools-guide-chromium/experimental-features/index.md

```diff
@@ -35,7 +35,7 @@ Microsoft Edge DevTools provide access to experimental features that are still i
 *  The latest version of the Canary preview channel of Microsoft Edge.
 *  The latest version of the Stable release of Microsoft Edge.
 
-All [channels of Microsoft Edge](/deployedge/microsoft-edge-channels) have experimental features. You can get the latest experimental features by using the [Microsoft Edge Canary channel](https://www.microsoftedgeinsider.com/welcome?channel=canary). To view the full list available in your version of Microsoft Edge, see the **Settings** > **Experiments** page in DevTools.
+All [channels of Microsoft Edge](/deployedge/microsoft-edge-channels) have experimental features. You can get the latest experimental features by using the [Microsoft Edge Canary channel](https://www.microsoft.com/edge/download/insider). To view the full list available in your version of Microsoft Edge, see the **Settings** > **Experiments** page in DevTools.
```

Comment:
existing url works atm: 
https://www.microsoftedgeinsider.com/welcome?channel=canary
goes to:
https://www.microsoftedgeinsider.com/en-us/welcome?channel=canary